### ISSUE TITLE
Create NUnit 3.0 documentation

### ISSUE THREAD

**Issue by CharliePoole** [nunit/nunit#168]
For NUnit 3.0, we need to decide the form our documentation is to take and implement it.

Documentation should use the same source for all distribution channels: web, download, man page, etc.

Documentation should be easily editable by contributors.

We need to have a design and plan, review it and then implement it.

**Comment by rprouse** [comment]
I read a blog post this morning about how GitHub uses GitHub pages to provide all the documentation for GitHub. Their documents are also versioned which would be useful for us. The blog post might give us some direction, https://github.com/blog/1939-how-github-uses-github-to-document-github

**Comment by CharliePoole** [comment]
Looking at the two alternatives we have been considering, it seems like the wiki is the simpler choice as compared to the use of GitHub pages. Additionally, if we should change our mind later, going from the  wiki to use of pages would involve less work in total than if we started out with pages and ended up going back to a wiki.

Therefore, here's the overall plan...

1. We'll continue updating the docs on the website through the alpha-5 release.
2. I'll create the overall structure of the documentation wiki on the nunit/nunit repository.
3. I'll set up a prioritized list of assignments for writing pages for the wiki and a process to track them.
4. I'll write some style guidelines for the documentation
5. We'll all start writing the docs as time permits.

**Comment by CharliePoole** [comment]
I'm closing this since the docs are now on the GitHub wiki. That's not to say they won't require further updates, but they currently have all info about 3.0 that was on the website. In fact, the wiki docs have been updated beyond what is on the website. 

I have changed the website link for the 3.0 docs to point to the wiki. In case they are needed, the older pages are still present, but not linked.
